dev-requirements.in

```diff
@@ -0,0 +1,2 @@
+pytest~=4.6
+pytest-cov~=2.7
```

Comment:
Do we prefer to set a version range here instead, like:
```
pytest>=4.6,<5
```

Reply:
I don't have a preference. The two are equivalent and this was slightly shorter... Do you have a preference? I am okay either way.

Reply:
Not really. I'm fine with both. I just remember the gdc dependency management wiki page stating that we should use ranges. If we're going to allow both, we should update the wiki page.